static/css/style.css

```diff
@@ -113,4 +112,126 @@ a {
   margin-bottom: 2rem;
   font-family: 'Roboto', sans-serif;
   font-size: 1.5rem;
+}
+
+@import url('https://fonts.googleapis.com/css2?family=Luckiest+Guy&family=Roboto:wght@700&display=swap');
+/*base*/
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+html {
+    font-size: 100%;   
+}
+
+a {
+    color: inherit;
+    text-decoration: none;
+}
+
+/*COMPONENTS*/
+/*button*/
+.error__button {
+    display: inline-block;  
+    padding: 0.5rem 0.6rem;
+    margin-bottom: 1rem;
+  
+    font-size: 1.2rem;
+    font-weight: bold;   
+    text-shadow: 0 0 0.2rem black;
+
+    border-radius: 2rem;
+    box-shadow: 0 0 0.15rem #FDF5BF;
+
+    cursor: pointer;
+}
+
+.error__button--notfound {
+    background-color: #FE5F55;
+    border: solid 0.1rem #FE5F55;
+}
+.error__button--notfound:hover {    
+    background-color: rgb(253, 65, 51);
+}
+
+.error__button--server {
+    background-color: #48A9A6;
+    border: solid 0.1rem #48A9A6;  
+}
+.error__button--server:hover {    
+    background-color: rgb(13, 131, 127);
+}
+
+/*LAYOUT*/
+/*container*/
+.error__container {  
+    max-width: 40rem;/*720px*/
+    margin: 0px auto;   
+}
+
+/*PAGES*/
+/*Error*/
+.error__box-img {    
+    background-image: url(img/logo-blond.svg);
+    background-repeat: no-repeat;
+    background-position: center center;
+    background-size: contain;
+}
+
+.error__heading {
+  margin-bottom: 2rem;
+  padding-top: 7rem;
+  font-size: 7rem;  
+}
+
+.error__text {
+  margin-bottom: 2rem;  
+  font-family: 'Roboto', sans-serif;
+  font-size: 1.5rem;
+  font-weight: bold;
+  text-shadow: 0 0 0.125rem black;
+}
+
+/*404*/
+.error-notfound {
+    height: 100vh;
+    color: #FE5F55;
+    font-family: 'Luckiest Guy', cursive;
+    text-shadow: 0 0 0.125rem #FDF5BF;
+    text-align: center;
+    background-color: #53131E;
+}
+
+/*500*/
+.error-server {
+    height: 100vh;
+    color: #48A9A6;
+    font-family: 'Luckiest Guy', cursive;
+    text-shadow: 0 0 0.125rem #FDF5BF;
+    text-align: center;
+    background-color: #003249;
+}
+
+/** Breakpoints */
+/** 1200px or smaller */
+@media (max-width: 75rem) {}
```

Comment:
If these are empty, they can be removed. Can you remove these please?

We may not need a lot of media queries so for now it's best to remove these empty media query declarations.